internal/controller/kustomization_controller.go

```diff
@@ -978,12 +987,18 @@ func (r *KustomizationReconciler) finalize(ctx context.Context,
 		!obj.Spec.Suspend &&
 		obj.Status.Inventory != nil &&
 		obj.Status.Inventory.Entries != nil {
+
 		objects, _ := inventory.List(obj.Status.Inventory)
 
+		statusPoller, pollingOpts, err := r.getPollerAndOptions(obj)
+		if err != nil {
+			return ctrl.Result{}, err
```

Comment:
Not sure if I should be returning an error here, why is the `inventory.List()` call right above swallowing the error?

Reply:
Yeah I think we should not even call `getPollerAndOptions` here and just use the default poller+options from the reconciler.